include/rocksdb/iterator.h

```diff
@@ -97,6 +97,8 @@ class Iterator : public Cleanable {
   // Property "rocksdb.iterator.super-version-number":
   //   LSM version used by the iterator. The same format as DB Property
   //   kCurrentSuperVersionNumber. See its comment for more information.
+  // Property "rocksdb.iterator.internal-key":
+  //   Get the internal key (e.g. tombstone) at which the iteration stopped.
```

Comment:
maybe at least clarify it's the user-key portion here, so user wouldn't know if it's tombstone or not (e.g., maybe it's an old version of a key covered by a tombstone)

Reply:
Updated the comment.